const promise = new Promise((resolve, reject) => {

    const x = 0
    if (x > 0) {
        reject()
    } else {
        resolve()
    }


})


promise
.then(() => console.log("ðŸš€ ~ promise resolved"))
.catch(() => console.log("ðŸš€ ~ promise REJECTED"))


// someFunction().then().catch()

